host_modules/debug_service.py

```
@@ -0,0 +1,146 @@
+import pty
+import subprocess
+import os
+import select
+import errno
+import logging
+
+from concurrent.futures import ThreadPoolExecutor
+from threading import Event
+
+from host_modules import host_service
+
+EXCEPTION_RAISED = 1
```

Consider why we need a exception code that is quite generic here. Can we return the exception's code?